common/utils.js

```diff
@@ -795,6 +795,24 @@ function validateDestinationChain(chains, destinationChain) {
     validateChain(chains, destinationChain);
 }
 
+async function calculateItsCrossChainGas(options) {
+    const { sourceChain, destinationChain, env, eventType } = options;
```

Comment:
confusing, better to use separate args than create your own object structure

Reply:
getting diverging recommendations here, see prev [comment](https://github.com/axelarnetwork/axelar-contract-deployments/pull/991#discussion_r2325724766). I'm open to either but just want to make sure we're consistent across the repo